test/cpp/end2end/test_service_impl.cc

```diff
@@ -444,16 +446,18 @@ experimental::ServerUnaryReactor* CallbackTestServiceImpl::Echo(
     }
 
     void StartRpc() {
-      if (req_->has_param() && req_->param().server_sleep_us() > 0) {
-        // Set an alarm for that much time
-        alarm_.experimental().Set(
-            gpr_time_add(gpr_now(GPR_CLOCK_MONOTONIC),
-                         gpr_time_from_micros(req_->param().server_sleep_us(),
-                                              GPR_TIMESPAN)),
-            [this](bool ok) { NonDelayed(ok); });
-      } else {
-        NonDelayed(true);
+      {
```

Comment:
These curly braces aren't needed anymore, right?

Reply:
Indeed, and removing them also reduces the size of this PR, so all is good!